src/verify/mod.rs

```diff
@@ -27,6 +27,18 @@ pub trait Credential {
     fn parse_claims(claims: ClaimsSet) -> Result<HashMap<String, ClaimValue>>;
 }
 
+pub fn decode_status_list(bits: u8, lst: String, idx: usize) -> u8 {
+    let status_list: JsonStatusList =
+        serde_json::from_str(format!("{{\"bits\": {}, \"lst\": \"{}\"}}", bits, lst).as_str())
+            .expect("failed to create status list");
```

Comment:
Would it make more sense to pass the `JsonStatusList` around as a string rather than the fields?